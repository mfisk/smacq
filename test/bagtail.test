((print -v srcip) + (print -v dstip)) from pcapfile(0325@1112-snort.pcap) 
