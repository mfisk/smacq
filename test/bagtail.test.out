srcip = 192.168.0.54
dstip = 192.168.0.210
srcip = 192.168.0.54
dstip = 192.168.0.210
srcip = 192.168.0.54
dstip = 192.168.0.210
srcip = 192.168.0.54
dstip = 192.168.0.210
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.200
dstip = 192.168.0.54
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
srcip = 192.168.0.54
dstip = 192.168.0.200
